README.md

```diff
@@ -229,19 +229,19 @@ These options follow the following rules:
 
 ## Templating
 
-It is possible to apply templates for status and tooltip. It allows you to customize the text of these inputs according to a particular template.
+JavaScript templates can be used to customize the status and tooltip.
```

Comment:
It is not a javascript template, or maybe I applied it without knowing about it ?

Reply:
I thought https://github.com/custom-cards/button-card#javascript-templates 😅

Reply:
Oo I didn't know about that template ahah, maybe for the future I will use this one but for the moment it is just a custom template.